policies/tiles.md

```diff
@@ -5,11 +5,9 @@ permalink: /policies/tiles/
 
 # {{ page.title }}
 
-We are in principle happy for our map tiles to be used by external users for creative and unexpected uses – in contrast to most web mapping providers, which insist that you use only their supplied API.
+**OpenStreetMap tiles are freely available for use as long as certain requirements are met by external users and enough resources are available from OpenStreetMap's own servers.** We are in principle happy for our map tiles to be used by external users for creative and unexpected uses – in contrast to most web mapping providers, which insist that you use only their supplied API.
```

Comment:
Could we be more specific here, and replace "certain requirements", which is very vague, with "the requirements below" or "the requirements on this page"?